plugins.json

```diff
@@ -2422,6 +2422,58 @@
     },
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-incident-response/",
```

Comment:
Not an issue given redirects, but we should update our `plugin.json` in v1.2 given the rename.

Reply:
agreed